bugsnag-android-core/src/main/java/com/bugsnag/android/AppData.java

```diff
@@ -120,17 +120,6 @@ String getActiveScreenClass() {
         return sessionTracker.getContextActivity();
     }
 
-    @NonNull
-    private String calculateNotifierType() {
-        String notifierType = config.getNotifierType();
-
-        if (notifierType != null) {
-            return notifierType;
-        } else {
-            return "android";
```

Comment:
This is the default value on the Configuration field now, so is unnecessary here